account_invoice_pricelist/views/account_invoice_view.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record id="view_account_invoice_customer_form" model="ir.ui.view">
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_form"/>
+        <field name="arch" type="xml">
+            <field name="user_id" position="after">
+                <group col="4" colspan="4">
+                    <field name="pricelist_id"/>
+                    <button type="object"
+                            name="button_update_prices_from_pricelist"
+                            string="Update prices"
+                            attrs="{'invisible': ['|', ('pricelist_id', '=', False), ('state', 'not in', ['draft'])]}"
+                            help="Update price in lines from the pricelist"
+                    />
+                </group>
+            </field>
+            <field name="invoice_line_ids" position="attributes">
+                <attribute name="context">{'partner_id': partner_id, 'price_type': context.get('price_type') or False, 'type': type, 'pricelist_id': pricelist_id}</attribute>
```

Comment:
Is this still needed?

Reply:
> You prefer, ok but do you use this module ?

@bealdav Yes, I made the v9 migration ;)

Reply:
ok thanks